### DISCUSSION TITLE
Classification Model_0 has no parameters

### DISCUSSION THREAD

**Discussion by avitalde** [mrdbourke/pytorch-deep-learning#788] [Q&A]
Video 66
Time: 9:36:30

I defined the following model
 ```
class CircleModelV0(nn.Module):
  def __init(self):
    super().__init__()
    self.layer_1 = nn.Linear(in_features=2, out_features=5) 
    self.layer_2 = nn.Linear(in_features=5, out_features=1)

  def forward(self, x):
    return self.layer_2(self.layer_1(x))

model_0 = CircleModelV0().to(device)
```

Now I want to check the model is on the correct device
<img width="646" alt="Screen Shot 2024-01-02 at 18 06 18" src="https://github.com/mrdbourke/pytorch-deep-learning/assets/74905217/bcdf38ff-90e1-4e59-a8c1-72a8f80257a6">

It seems like my model doesn't have any parameter and so I get an error. What can I do to fix this?

**Comment by AndrewRomitti** [comment]
Try creating an nn.Sequential in the class code. This is what pytorch official documentation suggests: https://pytorch.org/tutorials/beginner/basics/buildmodel_tutorial.html

So for your code:
`class CircleModelV0(nn.Module):`
`    def__init__(self):`
`       super().__init__()`
`        self.layers = nn.Sequential(`
`            nn.Linear(2,5)`
`            nn.Linear(5,1)`
`        )`
`    def forward(self, x):`
`        return self.layers(x)`

`model_0 = CircleModelV0().to(device)`

Let me know if this works! :)
